tasks/export_abi.js

```diff
@@ -54,6 +54,8 @@ subtask(
       abi = new Interface(abi).format(FormatTypes.minimal);
     } else if (config.format == "fullName") {
       abi = new Interface(abi).format(FormatTypes.fullName);
+    } else {
+      throw new HardhatPluginError(`Unknown format: ${config.format}`);
```

Comment:
What if `format === 'json'`?

Reply:
derp that should have been captured in L51 ! 
Sorry about that. Pushed in a fix